test/utils.js

```diff
@@ -57,6 +61,15 @@ const getPayloadAndProposalHash = async (commandID, target, nativeValue, calldat
     return [payload, proposalHash, eta];
 };
 
+const increaseTime = async (timeDelay) => {
+    if (isHardhat) {
```

Comment:
isHardhat is currently a function, and it isn't being called here. but doesn't need to be a function

Reply:
yup changed to bool